configs/IFLIGHT_R5_F435/config.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU                   AT32F435M
+
+#define BOARD_NAME                      IFLIGHT_R5_F435
+#define MANUFACTURER_ID                 IFRC
+
+// LEDS
+#define LED0_PIN                        PC15
+#define LED0_INVERTED
+#define BEEPER_PIN                      PB2
+#define BEEPER_INVERTED
+#define LED_STRIP_PIN                   PA8
+
+// MOTORS & SERVOS
+#define MOTOR1_PIN                      PB0
+#define MOTOR2_PIN                      PB1
+#define MOTOR3_PIN                      PB10
+#define MOTOR4_PIN                      PB11
+
+#define DEFAULT_DSHOT_BURST             DSHOT_DMAR_AUTO
+#define DEFAULT_DSHOT_BITBANG           DSHOT_BITBANG_ON
+
+// SPI
+#define SPI1_SCK_PIN                    PA5
+#define SPI1_SDI_PIN                    PA6
+#define SPI1_SDO_PIN                    PA7
+#define SPI1_NSS_PIN                    PA4
+
+#define SPI2_SCK_PIN                    PB13
+#define SPI2_SDI_PIN                    PB14
+#define SPI2_SDO_PIN                    PB15
+#define SPI2_NSS_PIN                    PB12
+
+#define SPI3_SCK_PIN                    PB3
+#define SPI3_SDI_PIN                    PB4
+#define SPI3_SDO_PIN                    PB5
+#define SPI3_NSS_PIN                    PA15
+
+// GYRO & ACC
+#define USE_ACC
+#define USE_GYRO
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO_SPI_ICM42688P
+#define GYRO_1_EXTI_PIN                 PC13
+#define ENSURE_MPU_DATA_READY_IS_LOW
+#define GYRO_1_CS_PIN                   SPI1_NSS_PIN
+#define GYRO_1_SPI_INSTANCE             SPI1
+#define GYRO_1_ALIGN                    CW0_DEG
+
+// OSD
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE            SPI2
+#define MAX7456_SPI_CS_PIN              SPI2_NSS_PIN
+
+// FLASH
+#define USE_FLASH
+#define USE_FLASH_W25M
+#define USE_FLASH_W25Q128FV             // Is this correct for W25Q512JVEIQ?
```

Comment:
This will need `USE_FLASH_W25M512` instead

https://github.com/betaflight/betaflight/blob/9730f9ee16ba645c157d3fdf0a02784ac4f99b2d/src/main/drivers/flash/flash_w25m.c#L52